scout/server/blueprints/institutes/controllers.py

```diff
@@ -131,6 +132,45 @@ def institute(store, institute_id):
     return data
 
 
+def populate_beacon_form(institute_obj):
+    """Populate the form select of scout.server.blueprints.institutes.forms.BeaconDatasetForm with data.
+    Available data is a controlled dictionary of dataset names depending of the datasets already existing in the Beacon server
+
+    Args:
+        institute_obj(dict) An institute object
+    """
+    beacon_form = BeaconDatasetForm()
+
+    if current_user.is_admin is False:
+        LOG.warning("User must be an admin to create a beacon dataset")
+        return beacon_form
+
+    if not all(
+        [current_app.config.get("BEACON_URL") and current_app.config.get("BEACON_TOKEN")]
+    ):  # Send a request to the beacon to collect the existing datasets
+        LOG.warning(
+            "Both BEACON_URL and BEACON_TOKEN params are required in Scout configuration file to create a new Beacon dataset"
+        )
+        return beacon_form
+
+    # Collect all dataset IDs names present on the Beacon
+    beacon_dsets = beacon.get_datasets()
+
+    dset_options = []  # List of tuples containing dataset select options
+    for build in ["GRCh37", "GRCh38"]:
```

Comment:
Are these already in use? If so, all good, they change slowly. Otherwise perhaps use and share a constant like BUILDS or a new one!

Reply:
These naming is used by the beacon app, not by Scout. In Scout we have often ['37','38']. Was this your question?

Reply:
https://github.com/ga4gh-beacon/specification/blob/b151e38e16644d9ae0bff048b859d03d96a34b7c/beacon.yaml#L147

Reply:
Thanks, it would not hurt to have a const with the official build ids, but the downside I guess could be that someone might use them for case build or such if they are too prominently global. As you wish - this will change very slowly so no biggie.

Reply:
It can be a constant in the beacon extension I guess, but I wouldn't add it in the scout constants because they are not used in other places in scout other than here

Reply:
Perfect - then they also won't be confused.